core/src/sumeragi/fault.rs

```diff
@@ -642,7 +642,7 @@ impl<G: GenesisNetworkTrait, F: FaultInjection> SumeragiWithFault<G, F> {
         );
         for event in Vec::<Event>::from(&block) {
             trace!(?event);
-            drop(self.events_sender.send(event));
+            let _send_event_result = self.events_sender.send(event);
```

Comment:
Same as above. Try printing some feedback, using e.g. `iroha_logger::debug!`.

Reply:
@QuentinI, you are right about failure reasons, [docs](https://docs.rs/tokio/latest/tokio/sync/broadcast/error/struct.SendError.html) says: 
> A send operation can only fail if there are no active receivers, implying that the message could never be received. The error contains the message being sent as a payload so it can be recovered.

> I'm guessing isn't a problem here

I agree that it's not strictly speaking problem, however i think we should at least log that send operation failed, it could be useful during debugging.

NOTE: it could be a problem because we are losing events